apps/component-date/fields/index.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+const dateComponent = require('hof-component-date');
+
+module.exports = {
+  name: 'component-date',
+  'date-of-birth': dateComponent('date-of-birth', {
+    // before means a date cannot be in the future
+    validate: ['required', 'date', 'before']
+  }),
+  'passport-expiry': dateComponent('passport-expiry', {
+    // after means a date cannot be in the past
+    validate: ['required', 'date', 'after']
```

Comment:
I shall now amend this to reflect the fact that the `date-component` has this by default